# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1434405147 14400
#      Mon Jun 15 17:52:27 2015 -0400
# Node ID 01895a6acdded4a1704c9437b228c78fede3c867
# Parent  bfe66db8addb63e53b460bc95d0cf91428caa5c6
* ov-oncleanup.h: Don't include config.h.

diff --git a/libinterp/octave-value/ov-oncleanup.h b/libinterp/octave-value/ov-oncleanup.h
--- a/libinterp/octave-value/ov-oncleanup.h
+++ b/libinterp/octave-value/ov-oncleanup.h
@@ -15,20 +15,16 @@ FITNESS FOR A PARTICULAR PURPOSE.  See t
 for more details.
 
 You should have received a copy of the GNU General Public License
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
 
-#ifdef HAVE_CONFIG_H
-#include <config.h>
-#endif
-
 #include <iosfwd>
 
 #include "ov-base.h"
 #include "ov-struct.h"
 #include "ov.h"
 
 static void
 gripe_internal (void)
